### PR TITLE
adding tooltip for compare button

### PR DESCRIPTION
**Pull request by californianseabass**
This change is [Reviewable](https://reviewable.io/reviews/sidewalklabs/ttx/56)

### CODE REVIEW

**Comment by californianseabass** [comment]
screen shot 2018-05-07 at 10 32 41 am

**Comment by californianseabass** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[packages/transit/static/index.html, line 22 at r1](https://reviewable.io/reviews/sidewalklabs/totx/56#-LBbGIyl7LUmC2NYEC2f:-LBv74AED2g13t5o-zAf:b-896fix) ([raw file](https://github.com/sidewalklabs/totx/blob/8cd6b13bbe64f1c8a90346e87e88fff3c7da56c4/packages/transit/static/index.html#L22)):*
<details><summary><i>Previously, danvk (Dan Vanderkam) wrote…</i></summary><blockquote>

Remove? You shouldn't need both this and the JSX.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/totx/56)*

File: packages/transit/src/travel-mode-selector.tsx [hunk 1]
```diff
@@ -106,7 +114,10 @@ export default class TravelModeSelector extends React.Component<Props, {}> {
           </>
         ) : (
           <div className="compare-button-wrapper">
-            <div className="compare-button" onClick={onCompare} />
+            <div className="compare-button" id="compare-button" onClick={onCompare} />
+            <div for="compare-button" className="mdl-tooltip mdl-tooltip--right hide-on-mobile">
```
Comment (danvk) [inline packages/transit/src/travel-mode-selector.tsx]: I think you want [`htmlFor`](https://reactjs.org/docs/dom-elements.html#htmlfor).

**Comment by californianseabass** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[packages/transit/src/travel-mode-selector.tsx, line 118 at r2](https://reviewable.io/reviews/sidewalklabs/totx/56#-LBb0oBRDt-spCLM51RB-r2-118:-LBvEEJ76TXr-_N9nw-O:b-896fix) ([raw file](https://github.com/sidewalklabs/totx/blob/a95c78a9a538384ab66533c3e72946953f718d00/packages/transit/src/travel-mode-selector.tsx#L118)):*
<details><summary><i>Previously, danvk (Dan Vanderkam) wrote…</i></summary><blockquote>

I think you want [`htmlFor`](https://reactjs.org/docs/dom-elements.html#htmlfor).
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/totx/56)*

**Comment by danvk** [comment]
:lgtm:

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---

*[packages/transit/src/travel-mode-selector.tsx, line 88 at r3](https://reviewable.io/reviews/sidewalklabs/totx/56#-LC0gZNZCH7I2p8C7wNe:-LC0gZNZCH7I2p8C7wNf:b1gg0ks) ([raw file](https://github.com/sidewalklabs/totx/blob/fe89acdb201a09ff475f61858a254a7db69b288f/packages/transit/src/travel-mode-selector.tsx#L88)):*
> ```XML
> }
> 
> declare module 'react' {
> ```

You should be able to remove this, `htmlFor` is standard.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/012108d8c6e2a28553c31fc1d4db0170a0cf90b4/types/react/index.d.ts#L1224

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/totx/56#-:-LC0hJmV2vzo1v9lKKqR:bnfp4nl)*

**Comment by californianseabass** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[packages/transit/src/travel-mode-selector.tsx, line 88 at r3](https://reviewable.io/reviews/sidewalklabs/totx/56#-LC0gZNZCH7I2p8C7wNe:-LC0jz4VBGfB-SzaZOwv:bepz96f) ([raw file](https://github.com/sidewalklabs/totx/blob/fe89acdb201a09ff475f61858a254a7db69b288f/packages/transit/src/travel-mode-selector.tsx#L88)):*
<details><summary><i>Previously, danvk (Dan Vanderkam) wrote…</i></summary><blockquote>

You should be able to remove this, `htmlFor` is standard.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/012108d8c6e2a28553c31fc1d4db0170a0cf90b4/types/react/index.d.ts#L1224
</blockquote></details>

tsc says otherwise: https://circleci.com/gh/sidewalklabs/totx/349

---


*Comments from [Reviewable](https://reviewable.io/reviews/sidewalklabs/totx/56)*

**Pull request merged**